rules/integrations/aws/privilege_escalation_sts_assumerole_usage.toml

```
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2021/05/17"
+maturity = "production"
+updated_date = "2021/10/05"
+integration = "aws"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies the use of AssumeRole. Attackers could move laterally and escalate privileges.
+"""
+false_positives = [
+    """
+    Automated processes that uses Terraform may lead to false positives.
+    """,
+]
+index = ["filebeat-*", "logs-aws*"]
+language = "kuery"
```

@austinsonger this one is missing the license
```suggestion
language = "kuery"
license = "Elastic License v2"
```